game/lobby.go

```diff
@@ -589,9 +598,16 @@ func createWordHintFor(word string, showAll bool) []*WordHint {
 				Underline: !irrelevantChar,
 			})
 		} else {
-			wordHints = append(wordHints, &WordHint{
-				Underline: !irrelevantChar,
-			})
+			if irrelevantChar {
```

Comment:
I have already done this. The reason it's still displayed here, is that I did it on the `develop` branch of the project. Branches are basically different versions of the project that can run in parallel. The `master` branch, on which you created this PR, contains the code deemed to be "stable and ready for public use", while "develop" is still not stable and therefore not ready for public use. All new changes should get into develop before getting into master. You can change this when you are creating a pull request. For now, don't worry, I'll simply change it onto develop myself.

Reply:
Yes, I saw that, sorry, it's just my setup was dirty, so I tried make a clean new repo for pull requests, and added that back to it, but not in correct order, and stuff, and I'm not a git-expert so..!

Reply:
Yeah, all of the concepts in git can be quite overwhelming in the beginning. I did tons of my mistakes myself ... well I still do sometimes. Most relevant concepts you need to understand are probably the following:

* Commit
* Branch
* Head
* Remote

As for commands:

* git branch
* git checkout
* git fetch
* git pull
* git push
* git add
* git commit
* git reset
* git log

These are the ones you usually use most. You don't need to know about every detail of them though. If you don't want to learn these thing right away, it's fine, you'll pick most of it up by running into problems and searching for a solution.